types/types.go

```diff
@@ -37,8 +37,7 @@ type ImageSource interface {
 	// May be "" if unknown.
 	IntendedDockerReference() string
 	// GetManifest returns the image's manifest.  It may use a remote (= slow) service.
-	// FIXME? This should also return a MIME type if known, to differentiate between schema versions.
-	GetManifest() ([]byte, error)
+	GetManifest() ([]byte, string, error)
```

Comment:
Please update the comment also to say what the second parameter is, and that "" means “unknown”.